src/routes/create-route-map-from-directory.ts

```diff
@@ -26,5 +26,11 @@ export const createRouteMapFromDirectory = async (
     }
   }
 
+  // Fix root route
+  if (routes[""]) {
+    routes["/"] = routes[""]
+    delete routes[""]
+  }
```

Comment:
how does this even happen? shouldn't `routeWithSlash` prevent this?

Reply:
no, because it replaces `/*` with `""`

but I think there might be a more straightforward solution, hang on

Reply:
is there a more elegant way to handle this?

Reply:
you could probably get clever with the regex so it doesn't capture the `/` when it's `/index.ts` (vs `/foo/index.ts`) but imo the current solution is probably a bit more readable than that

Reply:
K